source/documentBase.py

```diff
@@ -87,28 +114,31 @@ def _getTableCellAt(self,tableID,startPos,row,column):
 		raise NotImplementedError
 
 	_missingTableCellSearchLimit=3 #: The number of missing  cells L{_getNearestTableCell} is allowed to skip over to locate the next available cell
-	def _getNearestTableCell(self, tableID, startPos, origRow, origCol, origRowSpan, origColSpan, movement, axis):
+
+	def _getNearestTableCell(
+			self,
+			tableID,

```

Comment:
What is the type of this?

Reply:
I'm not sure, it's not documented anywhere. I spent some time investigating and couldn't get a clear idea. As the change is unrelated and no information is lost, I'm hesitant to spend too long investigating this.

Reply:
I had a quick look. In virtual buffers it is an int, whereas in UIA it is a tuple (the runtime id). I guess it's safe to mark it any, as long as it can be compared.

```suggestion
			# At least known to be a tuple for UIA and an integer for virtual buffers
			tableID, typing.Any,
```

Reply:
I added a type and summary to the module level, as `tableID` is used as an argument elsewhere in the file, and will need to get type information added eventually.